### ISSUE TITLE
gist:hasPhysicalLocation

### ISSUE THREAD

**Issue by uscholdm** [semanticarts/gist#940]
```
gist:hasPhysicalLocation
	a
		owl:ObjectProperty ,
		owl:TransitiveProperty
		;
	rdfs:range gist:Place ;
	skos:definition "Where something is located"^^xsd:string ;
	skos:prefLabel "has physical location"^^xsd:string ;
	.
```

I propose changing the skos definition to:
- "Relates something to where it is physically located."

**ALSO**: 
I propose adding a subclass restriction to the class, `gist:PhysicalIdentifiableItem` requiring it to have a physical location. Ie. `(gist:hasPhysicalLocation some gist:Place)`. Perhaps the same for `gist:PhysicalSubstance`

**Comment by rjyounes** [comment]
How about: "Relates something to its physical location"?  

The restriction seems good.

Didn't we also want a scope note to the effect that the location could be either permanent (a house) or transient (a car), and that if someone wants to specify a transient relationship they should use a temporal relation?

**Comment by uscholdm** [comment]
> How about: "Relates something to its physical location"?

A definite improvement. Good idea.

**Comment by uscholdm** [comment]
> Didn't we also want a scope note to the effect that the location could be either permanent (a house) or transient (a car), and that if someone wants to specify a transient relationship they should use a temporal relation?

We could, in which case, how about:
```
skos:scopeNote "To model changing locations, create a subclass of TemporalRelation (e.g. EstablishedLocation)" that reifies hasPhysicalLocation^^xsd:string ; 
```
This might be setting a precedent; we don't usually say how to extend gist, that could get out of hand quickly. But it might make sense in this case. Let's see what people think.

**Comment by uscholdm** [comment]
### Updated proposal
```
gist:hasPhysicalLocation
	a
		owl:ObjectProperty ,
		owl:TransitiveProperty
		;
	rdfs:range gist:Place ;
	skos:definition "Relates something to its physical location"^^xsd:string ;
	skos:prefLabel "has physical location"^^xsd:string ;
	.
```

**Comment by rjyounes** [comment]
Oddly, `EstablishedLocation` sounds to me like the permanent version.

Maybe we don't have to give explicit suggestions. How about just:

"This property includes both temporary (e.g., a car) and permanent (e.g., a house) locations of objects."

**Comment by rjyounes** [comment]
Closing - fixed in PR #941.
